distributed/protocol/serialize.py

```diff
@@ -24,6 +25,7 @@
     pack_frames_prelude,
     unpack_frames,
 )
+from distributed.sizeof import safe_sizeof as sizeof
```

Comment:
I think you can use dask.sizeof for everything.
safe_sizeof should be just a wrapper at the top level, e.g. when directly invoked by the user instead of deep in the recursion.

Reply:
`test_dataframe_attribute_error` failed when using the plain sizeof but you are probably right, this should be used somewhere further up the stack

Reply:
got it. the to_frames should've been using the safe_sizeof